pkgs/by-name/ss/ssh-import-id/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  extraHandlers ? [ ],
+  fetchgit,
+  installShellFiles,
+  makeWrapper,
+  python3,
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
This shouldn use python3.pkgs

Also is buildPythonApplication right? Why not buildPythonPackage?

Reply:
> This shouldn use python3.pkgs

I guess that the slicing issue is back as the manual indeed says to use `python3Packages`. Fixed.

> Also is buildPythonApplication right? Why not buildPythonPackage?

`buildPythonPackage` is for Python module. This is an application which is not imported by other modules.